builder/antlr/deps.bzl

```diff
@@ -3,8 +3,8 @@ load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 def deps():
     git_repository(
         name = "rules_antlr",
-        remote = "https://github.com/marcohu/rules_antlr",
-        tag = "0.5.0"
+        remote = "https://github.com/vaticle/rules_antlr",
```

Comment:
Until https://github.com/marcohu/rules_antlr/pull/18 is merged, we need to use our own fork with Rust support